config/mbed_config.h

```diff
@@ -21,6 +21,8 @@
 #include "rover_config.h"
 
 // MBED configuration parameters
+#define CLOCK_SOURCE                                                          USE_PLL_HSE_EXTC|USE_PLL_HSI                                                                     // set by target:NUCLEO_F446RE
```

Comment:
Are these supposed to be here?

Reply:
no they arent, idk why id didnt have a mergo conflict

Reply:
pls checj this before merging

Reply:
sounds good. i will delete the 2 lines of code and push a new version